tests/function/test_messagets.py

```diff
@@ -60,10 +60,11 @@ def test_items(self):
             },
         ]
         msgts = MessageTs(*mock_obj)
+        self.assertEqual(len(mock_obj), len(msgts))
         self.assertEqual(len(mock_obj), len(msgts.items()))
```

Comment:
remove line 64?

Reply:
Kept for coverage, and making sure that items actually return the values

Reply:
Do we still need the items() method now that we can use `msgts[0]` and `msgs[0]`? If we remove items(), coverage will be fine. Same for above.

Reply:
I was thinking the same. Maybe not. @vigith what do you think? Maybe I can add a deprecated message, because otherwise this will be a breaking change.

Reply:
if we are using it then deprecate it now and completely remove it in a later release, else remove it now.